### PR TITLE
Volume device type

### PR DESCRIPTION
**Pull request by ferullo**
## Change Summary

This adds `volume_device_type`.

### Sample values

Sample document:

None

## Release Target

Ideally for 8.4

## Q/A

N/A

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes
- [ ] If these field(s) are "exception"-able, I made a companion PR to Kibana adding it (see [Readme](https://github.com/elastic/endpoint-package#exceptionable))
- [ ] If this is a `metadata` change, I also updated both transform destination schemas to match

### For Transform changes:

- [ ] The new transform successfully starts in Kibana
- [ ] The corresponding transform destination schema was updated if necessary

### CODE REVIEW

File: custom_schemas/custom_dll.yml [hunk 1]
```diff
@@ -171,6 +171,13 @@
       description: >
         VendorID of the device. It is provided by the vendor of the device.
 
+    - name: Ext.device.volume_device_type
+      level: custom
+      type: keyword
+      short: FILL ME IN
```
Comment (ferullo) [inline custom_schemas/custom_dll.yml:177]: @Trinity2019 can you update these FILL ME INs

**Comment by Trinity2019** [comment]
Thanks @ferullo for updating endpoint-package!

**Comment by ferullo** [comment]
Thanks for the doc updates @Trinity2019 . For future reference, I should have mentioned that only the files in `custom_schemas/` needed to be updated. The rest are auto generated files (which nevertheless are suppose to be committed).

**Comment by ferullo** [comment]
@elastic/security-onboarding-and-lifecycle-mgt can this be merged or is there a merge-freeze right now?

**Comment by kevinlog** [comment]
@ferullo we have just cut the Endpoint package over for `8.4.0` and released it.  Is this critical for `8.4.0` or can it wait for the following release?

Ideally, we start to enforce merge-freezes a couple days before stack FF so that we have time to release the package and ensure there aren't any testing issues.

@pzl can correct me if I'm wrong, we can release another package for `8.4.0`, but we'll need to increment the version and re-release the package.

**Comment by ferullo** [comment]
These fields will be generated by Endpoint in 8.4 but it won't cause any problems if we don't merge this. Waiting for the next package release is ok.

**Comment by pzl** [comment]
When this gets in, we can backport for an `8.4.1`

**Comment by ferullo** [comment]
@pzl can this be merged and included in 8.4.1 along with https://github.com/elastic/endpoint-package/pull/280 ?

**Comment by pzl** [comment]
@ferullo 

Yes, this and #280 will be `8.4.1` release as soon as we get them both in.


For this PR, can we update the test files with values for this field?  Looks like that should be:

- `package/endpoint/data_stream/alerts/sample_event.json` adding:
  + `threat.enrichments.indicator.file.Ext.device.volume_device_type`
  + `threat.indicator.file.Ext.device.volume_device_type` 
- `package/endpoint/data_stream/file/sample_event.json` adding:
  + `file.Ext.device.volume_device_type`
- `package/endpoint/data_stream/library/sample_event.json` adding:
  + `dll.Ext.device.volume_device_type`
- `package/endpoint/data_stream/process/sample_event.json` adding:
  + `process.Ext.device.volume_device_type`


Once we have values in the test files, this is good to merge

**Comment by ferullo** [comment]
@Trinity2019 This PR doesn't have `file.Ext.device.volume_device_type` appear in alert documents. Is it correct to leave it out of alerts? Endpoint testing passes without it in alerts so on the one hand it must not be needed. But I'm not sure if that is because of a quirk in Endpoint testing or if it really never would appear in alerts.

**Comment by pzl** [comment]
I believe the CI failure is just the newlines at the end of the sample_event.json files.

**Comment by ferullo** [comment]
Ah thanks, fixed. I did not make the change you suggested for `package/endpoint/data_stream/alerts/sample_event.json`. I'm not sure why this new field is appearing in that doc under `threat_intel`. I assume there is some `*` used that is pulling in all fields, it isn't intentionally added under `threat_intel`.

**Comment by ferullo** [comment]
@pzl are you ok with this being merged?

**Review by pzl — APPROVED** [review]
looks good. ok to merge. I will issue a followup PR that will remove 

- `threat.enrichments.indicator.file.Ext.device.volume_device_type`
- `threat.indicator.file.Ext.device.volume_device_type`

from the alerts index before we ship anywhere

**Comment by ferullo** [comment]
OK, thanks @pzl!

**Pull request merged**